crispy_tailwind/templates/tailwind/layout/select.html

```diff
@@ -1,9 +1,9 @@
 {% load crispy_forms_filters %}
-{% load tailwind_filters %}
+{% load tailwind_filters tailwind_field %}
 {% load l10n %}
 
 <div class="relative">
-    <select class="bg-white focus:outline-none border {% if field.errors %}border-red-500 {% else %}border-gray-300 {% endif %}rounded-lg py-2 px-4 block w-full appearance-none leading-normal text-gray-700" name="{{ field.html_name }}" {{ field|build_attrs }}>
+    <select class="{{ field|tailwind_field_class }}{% if field.errors %} border-red-500{% endif %}" name="{{ field.html_name }}" {{ field|build_attrs }}>
```

Comment:
praise: that sounds like a solid idea! 👌

Reply:
question: why did you only change the select field, shouldn't this apply to all elements? 🤔

Reply:
ah, it's just a starting point?

Reply:
Yes, it's a working starting point that can be applied to other elements in the future, once this has been accepted and merged, i can continue to add this feature to other elements too

Reply:
Ok, quick follow-up question: The logic changed from If/else for red or grey to put red. Will red overwrite grey in the error case?